docs/preview/features/correlation.md

```diff
@@ -108,4 +108,64 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 }
 ```
 
+## Using secret store within Azure Functions
+
+### Installation
+
+For this feature, the following package needs to be installed:
+
+```shell
+PM > Install-Package Arcus.WebApi.Logging.AzureFunctions
+```
+
+### Usage
+
+To make sure the correlation is added to the HTTP response, following additions have to be made in the `.Configure` methods of the function's startup:
+
+```csharp
+[assembly: FunctionsStartup(typeof(Startup))]
+
+namespace MyHttpAzureFunction
+{
+    public class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            builder.AddHttpCorrelation();
+        }
+    }
+}
+```
+
+When this addition is added, you can use the `HttpCorrelation` inside your function to call the correlation functionality and use the `ICorrelationInfoAccessor` (like before) to have access to the `CorrelationInfo` of the HTTP request.
+
+```csharp
+public class MyHttpFunction
+{
+    private readonly HttpCorrelation _correlationService;
+
+    public MyHttpFunction(HttpCorrelation correlationService)
```

Comment:
maybe rename `correlationService?`

Reply:
Yes, renamed to `CorrelationService correlationService`.

Reply:
Actually I just meant the variable name, not the type name as HttpCorrelation is what it's doing if you ask me.

I would revert the renaming from https://github.com/arcus-azure/arcus.webapi/pull/171/commits/3e3a7011d206cf2658aba3f263fb197c3cb22f16

Reply:
😅 Yeah, I already thought there was something off.

Reply:
Great, but can you still rename the variable please? :D